_posts/pt_BR/2016-01-03-improve-nested-conditionals.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+
+title: Melhorar Condicionais Aninhados
+tip-number: 03
+tip-username: AlbertoFuente 
+tip-username-profile: https://github.com/AlbertoFuente
+tip-tldr: Como podemos melhorar e tornar mais eficiente a declaração de `if` aninhados em javascript?
+
+categories:
+    - pt_BR
+---
+
+Como podemos melhorar e tornar mais eficiente a declaração de `if` aninhados em javascript?
+
+```javascript
+if (color) {
+  if (color === 'black') {
+    printBlackBackground();
+  } else if (color === 'red') {
+    printRedBackground();
+  } else if (color === 'blue') {
+    printBlueBackground();
+  } else if (color === 'green') {
+    printGreenBackground();
+  } else {
+    printYellowBackground();
+  }
+}
+```
+
+Uma forma de melhorar declarações de `if` aninhados seria usar `switch`. Embora seja menos verboso e mais ordenado, o uso não é recomendado porque seu debug é muito difícil. [Aqui](https://toddmotto.com/deprecating-the-switch-statement-for-object-literals) o porquê.
```

Comment:
Here I have never seen someone saying "verboso", only "verbose". Not sure if there is a standard or what could be better in this case.